myuw/urls.py

```diff
@@ -118,14 +119,24 @@
         login_required(InstScheQuar().run),
         name="myuw_instructor_schedule_api"
         ),
+    url(r'^api/v1/instructor_section/(?P<year>\d{4}),(?P<quarter>[a-zA-Z]+),'
+        r'(?P<curriculum>[\w& ]+),(?P<course_number>\d{3})\/'
+        r'(?P<course_section>[A-Z][A-Z0-9]?)$',
+        login_required(InstSect().run),
+        name="myuw_instructor_section_api"
+        ),
     url(r'^api/v1/thrive/$', login_required(ThriveMessages().run),
         name="myuw_thrive_api"
         ),
     url(r'^choose/new', new_site, name="myuw_pref_new_site"
         ),
     url(r'^choose/legacy', old_site, name="myuw_pref_old_site"
         ),
-    url(r'^teaching/?', teaching, name="myuw_teaching_page"
+    url(r'^teaching/?$', teaching, name="myuw_teaching_page"
+        ),
+    url(r'^teaching/'
+        r'(?P<section>\d{4},[a-zA-Z]+,[\w& ]+,\d{3}\/[A-Z][A-Z0-9]?)$',
```

Comment:
gotcha, i see the canvas courses now w/ multiple students in a per-instructor independent study section.